### PR TITLE
Malin V & Jessikas Labyrinth app

### PR DESCRIPTION
**Pull request by malinvan**
Link to my deployed project:
https://malin-and-jessikas-labyrinth.netlify.app

My reflections on how this week's project turned out:
See README

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
+  game: game.reducer,
+  ui: ui.reducer
+})
+
+const store = configureStore({ reducer })
+
+export const App = () => (
+  <Provider store={ store }>
+    <Labyrinth />
+    <Main />
```
Comment (jesinkey) [inline code/src/App.js:20]: Nice solution with the background as a component, its a great detail!

File: code/src/components/Labyrinth.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import styled, { keyframes } from 'styled-components';
+
+const card = keyframes`
```
Comment (jesinkey) [inline code/src/components/Labyrinth.js:4]: Awesome use of styled components, the code it really easy to follow

File: code/src/components/Loading.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import styled, { keyframes } from 'styled-components';
```
Comment (jesinkey) [inline code/src/components/Loading.jsx:2]: Very nice loading page, impressive that you created the Icon yourself and didnt use an image!

File: code/src/components/Styling.js [hunk 4]
```diff
@@ -0,0 +1,175 @@
+import styled, { keyframes } from 'styled-components';
```
Comment (jesinkey) [inline code/src/components/Styling.js:1]: I really like this solution of using a separate component for some "main" styling. Must make work quicker and easier

**Review by jesinkey — APPROVED** [review]
I have nothing negative to say! Your project looks great and the code was super easy to follow. I will be stealing some ideas for the future =)

File: code/src/App.js [hunk 5]
```diff
@@ -1,9 +1,22 @@
 import React from 'react'
+import { Provider } from "react-redux";
+import { combineReducers, configureStore} from "@reduxjs/toolkit";
+import { game } from "./reducers/game";
+import { ui } from './reducers/ui';
```
Comment (KarolCislowski) [inline code/src/App.js:5]: remember to make one empty line before your reducers as well 😸

File: code/src/components/Labyrinth.js [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+`;
+
+export const Labyrinth = () => {
+  return (
+    <Container>
+      <BorderRight></BorderRight>
+      <BorderBottom></BorderBottom>
+      <BorderBottom></BorderBottom>
+      <SmallBox></SmallBox>
+      <BorderBottom></BorderBottom>
```
Comment (KarolCislowski) [inline code/src/components/Labyrinth.js:57]: These Components are empty, so why didn't you use `<ComponentName />` notation?

File: code/src/components/Loading.jsx (hunk 3 above)
Comment (KarolCislowski) [inline code/src/components/Loading.jsx:2]: Be consistent with semicolons 🤓

File: code/src/components/Loading.jsx [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+  font-size: 22px;
+  margin-top: 20px;
+  color: white;
+`;
+
+export const Loading = () => {
+  return (
+    <Container>
+      <Compass>
+        <Needle></Needle>
```
Comment (KarolCislowski) [inline code/src/components/Loading.jsx:64]: Again empty Component with a separate closing tag.

**Review by KarolCislowski — APPROVED** [review]
It is a very nice app 😸  Remember to use self-closing tags when possible 😸 
I really like the maze background 🥰

**Pull request closed without merging**